importer/main.py

```diff
@@ -861,8 +862,75 @@ def clean_duplicates(users, cascade_delete):
+        csv_writer.writerows(data)
+
+
+# This function exports resources from the API to a csv file
+def export_resources_to_csv(resource_type, parameter, value, batch_size):
+    resource_type = get_valid_resource_type(resource_type)
+    resource_url = "/".join([config.fhir_base_url, resource_type])
+    if len(parameter) > 0:
+        resource_url = resource_url + "?" + parameter + "=" + value + "&_count=" + str(batch_size)
+    response = handle_request("GET", "", resource_url)
+    if response[1] == 200:
+        resources = json.loads(response[0])
+        data = []
+        if resource_type == "Location":
+            elements = ["name", "status", "id", "identifier", "parentName", "parentID", "type", "typeCode",
+                        "physicalType", "physicalTypeCode"]
+        elif resource_type == "Organization":
+            elements = ["name", "active", "id", "identifier", "alias"]
+        elif resource_type == "CareTeam":
+            elements = ["name", "status", "id", "identifier", " organizations", "participants"]
+        else:
+            elements = []
+        for x in resources["entry"]:
+            rl = []
+            for element in elements:
+                try:
+                    if element == "active":
+                        value = x["resource"]["active"]
+                    elif element == "identifier":
+                        value = x["resource"]["identifier"][0]["value"]
+                    elif element == "parentName":
+                        value = x["resource"]["partOf"]["display"]
+                    elif element == "parentID":
+                        value = x["resource"]["partOf"]["reference"]
+                    elif element == "type":
+                        value = x["resource"]["type"][0]["coding"][0]["display"]
+                    elif element == "typeCode":
+                        value = x["resource"]["type"][0]["coding"][0]["code"]
+                    elif element == "physicalType":
+                        value = x["resource"]["physicalType"]["coding"][0]["display"]
+                    elif element == "physicalTypeCode":
+                        value = x["resource"]["physicalType"]["coding"][0]["code"]
+                    elif element == "alias":
+                        value = x["resource"]["alias"][0]
+                    else:
+                        value = x["resource"][element]
+                except KeyError:
+                    value = ""
+                rl.append(value)
+            data.append(rl)
+        write_csv(data, resource_type, elements)
+    else:
+        logging.error(f"Failed to retrieve resource. Status code: {response[1]}")
+
+
 @click.command()
-@click.option("--csv_file", required=True)
+@click.option("--csv_file", required=False)
+@click.option("--export_resources", required=False)
+@click.option("--parameter", required=False)
+@click.option("--value", required=False)
+@click.option("--batch_size", required=False)
```

Comment:
this batch_size needs to be changes to `limit`

Reply:
I label it as `limit` and not `batch_size` ?